Data/ShopContext.cs

```diff
@@ -28,5 +28,12 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             modelBuilder.Entity<ShoppingCart>().ToTable("ShoppingCart");
             modelBuilder.Entity<Discount>().ToTable("Discount");
         }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            optionsBuilder.UseLazyLoadingProxies();
+
+            base.OnConfiguring(optionsBuilder);
+        }
```

Comment:
What these changes do?

Reply:
I tried using lazy loading for fetching database data.